src/components/Login/Login.jsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import Header from "./Header/Header";
+import MainContent from "./MainContent/MainContent";
+
+const Login = ({ onThemeToggle, mode, theme }) => {
+  return (
+    <>
+      <Header onThemeToggle={onThemeToggle} mode={mode} />
```

Comment:
Why header is here ? if its common across your pages, the put it outside route then

Reply:
no header is not common for login and home page @faran4engg